### DISCUSSION TITLE
Image within sidebar overflows the width when set to 100%

### DISCUSSION THREAD

**Discussion by ma-sadeghi** [pradyunsg/furo#119] [Potential Issue] [answered]
Here's part of my `index.rst`:
```rst
.. sidebar:: Highlight

    The animation below shows the end result of an image-based invasion
    percolation algorithm developed in PMEAL and implemented in PoreSpy.
    Unlike physics-based approaches like the Lattice-Boltzmann method, our
    approach is very quick and almost runs instantly on average-size
    images.

    .. image:: _static/images/image_based_ip.gif
       :width: 100%
       :align: center
```
and here's how it renders:
![image](https://user-images.githubusercontent.com/14086031/113309979-b65a6980-92d5-11eb-9a97-1ab46e30d53b.png)

**Comment by ma-sadeghi** [comment, marked as answer]
I just found a fix. Instead of using `image` directive, I can use `figure` directive with `figwidth` attribute, which works fine without overflowing the container.
```rst
.. figure:: _static/images/image_based_ip.gif
   :figwidth: 100%
   :align: center
```

**Reply by pradyunsg** [reply]
Using `:width: 100%` injects CSS into the generated page such that I couldn't ever figure out how to make work right. Using figures works, because they get wrapped and that wrapper "just does the right thing".

I wish this were easier, but hey, I'm glad you found the workaround. :)
